### PR TITLE
SDL readme

### PR DESCRIPTION
**Pull request by leogermond**
Add a README for the `extern/` directory, pointing to resources for installing the SDL library and wrappers.

### CODE REVIEW

File: extern/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+`gprbuild -Xsdl_backend=gnat_sdl2 project.gpr`
+
+Examples of the use can be found in [`courses/fundamentals_of_ada/labs/solar_system`](https://github.com/AdaCore/training_material/tree/master/courses/fundamentals_of_ada/labs/solar_system).
+
+You can access a properly packaged version of those in the
+[repository Actions page](https://github.com/AdaCore/training_material/actions).
+
+Validate by clicking the checkmark on the side.
+
+In case of further issues, try changing the `backend` option to other values. Keep
```
Comment (QuentinOchem) [inline extern/README.md:73]: This should probably be a different section "Troubleshooting" where we may actually have a few more entries.

File: extern/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+![For SDL2, select gnat_sdl2 as sdl_backend](../images/gnat_studio/scenario/sdl_backend_sdl2.png)
+
+For manual GPR Build, use the `-Xsdl_backend=SDL2` option
+
+`gprbuild -Xsdl_backend=gnat_sdl2 project.gpr`
+
+Examples of the use can be found in [`courses/fundamentals_of_ada/labs/solar_system`](https://github.com/AdaCore/training_material/tree/master/courses/fundamentals_of_ada/labs/solar_system).
+
+You can access a properly packaged version of those in the
```
Comment (QuentinOchem) [inline extern/README.md:68]: This is a bit confusing to me. What is properly packaged, and what do we mean by that? As a gihub user but non-expert, it wasn't obvious to me either how to access it. Let me suggest to either create a separate section with more details, or remove that hint.

**Review by QuentinOchem — CHANGES_REQUESTED** [review]

File: extern/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+# SDL install
+
+## Windows
+
+For Windows users, you must download the DLL for the following libraries
+
+* [SDL2](https://www.libsdl.org/download-2.0.php).
+* [SDL2 ttf, zlib, and libfreetype](https://www.libsdl.org/projects/SDL_ttf/)
+* Extract all the DLL file to a the target directory
```
Comment (frank-at-adacore) [inline extern/README.md:20]: file => files

File: extern/README.md (hunk 2 above)
Comment (leogermond) [reply extern/README.md:68]: I removed it, it is not needed to distribute and use the labs. We'll update that when the packages are actually deployed.

File: extern/README.md (hunk 1 above)
Comment (leogermond) [reply extern/README.md:73]: Agreed, makes sense

**Review by QuentinOchem — APPROVED** [review]

**Pull request merged**